test/steps/image_registry.go

```diff
@@ -141,16 +150,26 @@ func getKogitoImageTag(projectLocation, mavenProfiles string, mavenOptions []str
 
 // Returns Kogito image name in the form of "<project location base>(-<maven profile>)*(-<maven option>)*"
 func getKogitoImageName(projectLocation, mavenProfiles string, mavenOptions []string) string {
+	var kogitoApplicationNameParts []string
+
+	if runtimeApplicationImageNamePrefix := config.GetRuntimeApplicationImageNamePrefix(); len(runtimeApplicationImageNamePrefix) > 0 {
+		kogitoApplicationNameParts = append(kogitoApplicationNameParts, runtimeApplicationImageNamePrefix)
+	}
+
 	kogitoApplicationBaseName := filepath.Base(projectLocation)
-	kogitoApplicationNameParts := []string{kogitoApplicationBaseName}
+	kogitoApplicationNameParts = append(kogitoApplicationNameParts, kogitoApplicationBaseName)
 
 	if len(mavenProfiles) > 0 {
 		mavenProfilesSlice := strings.Split(mavenProfiles, ",")
 
 		// Sort profiles to generate consistent image name
 		sort.Strings(mavenProfilesSlice)
 
-		kogitoApplicationNameParts = append(kogitoApplicationNameParts, mavenProfilesSlice...)
+		for _, profile := range mavenProfilesSlice {
+			if profile != "default" {
```

Comment:
Nitpick: AFAIK the `default` profile there was just a placeholder. Maybe it would be better to remove the `default` profile usage than creating workarounds, what do you think?

Reply:
true ... will update the syntax for default profile

Reply:
syntax updated and this part removed. @sutaakar please let me know

Reply:
Looks good now, thanks